models.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import Column, Integer, String, Text, ForeignKey, Date
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class DBAuthor(Base):
+    __tablename__ = "authors"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(255), unique=True)
+    bio = Column(Text)
+
+    books = relationship("DBBook", back_populates="author")
+
+
+class DBBook(Base):
+    __tablename__ = "books"
+
+    id = Column(Integer, primary_key=True, index=True)
+    title = Column(String(255), unique=True)
+    summary = Column(Text)
+    publication_date = Column(Date)
```

Comment:
so it is possible to set any date for publication, even if it is future?
Also, there can be no publication date at all, but with the existing author!?